modules/git/commit_archive.go

```diff
@@ -20,18 +20,43 @@ const (
 	TARGZ
 )
 
-// CreateArchive create archive content to the target path
```

Comment:
Could you add Gitea copyright to header of this file?

Reply:
done